packages/eslint-plugin/src/utils/create-no-tagged-template-expression-rule/index.ts

```diff
@@ -39,11 +39,18 @@ export const createNoTaggedTemplateExpressionRule =
             return;
           }
 
+          const args = toArguments(source, quasi);
+
+          // Skip invalid CSS
+          if (args.length < 1) {
```

Comment:
RFC: This is to prevent eslint rule to break when dealing with invalid CSS such as `margin-top 0;` ( missing colon symbol).
However, maybe we should throw an error instead.

Reply:
Throwing an error will cause the lint rule to fail. This shouldn't be an error for the lint rule as it's expected to see invalid CSS in the AST. 

We should skip auto-fixing if we would end up with invalid code if transformed.

Reply:
absolutely right re auto-fixing!